examples/js/loaders/LWOLoader.js

```diff
@@ -0,0 +1,2304 @@
+/**
+ * @author Lewy Blue https://github.com/looeee
+ *
+ * Load files in LWO3 format
+ *
+ * LWO3 format specification:
+ * 	http://static.lightwave3d.com/sdk/2018/html/filefmts/lwo3.html
+ *
+ * LWO2 format specification (not supported by this loader, however LWO3 files contain a lot from the LWO2 spec)
+ * 	http://static.lightwave3d.com/sdk/2018/html/filefmts/lwo2.html
+ *
+ */
+
+THREE.LWOLoader = ( function () {
+
+	var lwoTree;
+
+	function LWOLoader( manager ) {
+
+		this.manager = ( manager !== undefined ) ? manager : THREE.DefaultLoadingManager;
+
+	}
+
+	LWOLoader.prototype = {
+
+		constructor: LWOLoader,
+
+		crossOrigin: 'anonymous',
+
+		load: function ( url, onLoad, onProgress, onError ) {
+
+			var self = this;
+
+			var path = ( self.path === undefined ) ? THREE.LoaderUtils.extractUrlBase( url ) : self.path;
+
+			// give the mesh a default name based on the filename
+			var modelName = url.split( path ).pop().split( '.' )[ 0 ];
+
+			var loader = new THREE.FileLoader( this.manager );
+			loader.setResponseType( 'arraybuffer' );
```

Comment:
It seems you are not using `loader.setPath()` here. This should be done here so the loader is more consistent to the others.

Reply:
Good catch!